app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -174,7 +174,7 @@ def check_policy_prevent(policy_event, event_target, cb_method)
       :class_name  => self.class.to_s,
       :instance_id => id,
       :method_name => :check_policy_prevent_callback,
-      :args        => [cb_method, event_target],
+      :args        => [cb_method, event_target.class.name, event_target.id],
```

Comment:
> P.S. I wish there was a way to actually prevent AR serializion, instead of constrained coding style that "dictates and protects"... I know it's a hard problem.

would having a (dev mode?) functionality of preventing args of a type (application record?) from going in the queue work?